pkg/sync/v5/data/Service/default/router.yaml

```diff
@@ -16,4 +16,5 @@ spec:
     port: 443
   selector:
     router: router
+  sessionAffinity: ClientIP
```

Comment:
think you're going to need to add
```
  sessionAffinityConfig:
    clientIP:
      timeoutSeconds: 10800
```
here as well - hopefully as it stands this should fail the sync pod test

Reply:
I read in the docs that the default for the timeoutSeconds is 10800 if `sessionAffinity: ClientIP` is set.  If you'd prefer I'll add this setting to be explicit.

Reply:
(Link to the docs which state these defaults for posterity's sake: https://v1-11.docs.kubernetes.io/docs/concepts/services-networking/service/#virtual-ips-and-service-proxies)

> In any of these proxy model, any traffic bound for the Service’s IP:Port is proxied to an appropriate backend without the clients knowing anything about Kubernetes or Services or Pods. Client-IP based session affinity can be selected by setting `service.spec.sessionAffinity` to “ClientIP” (the default is “None”), and you can set the max session sticky time by setting the field `service.spec.sessionAffinityConfig.clientIP.timeoutSeconds` if you have already set `service.spec.sessionAffinity` to “ClientIP” (the default is “10800”).

Reply:
@jim-minter, you were right. Ran a local test and the sync pod was adding these settings.   I have updated PR